### PR TITLE
AT-4219 Fixing tasking order translate bug

### PR DESCRIPTION
**Pull request by andher1802**
Fixing the OrderV2 migration to allow tasking orders.

Items:
* [ ] Ran test & live-tests
* [ ] Implemented (new) unit tests
* [ ] Removed credentials
* [ ] Removed argument pointing to staging
* [ ] Updated [documentation](sdk.up42.com)

For release:
* [ ] Bumped version
* [ ] Added changelog
* [ ] Updated announcement banner

### CODE REVIEW

File: tests/test_order.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import pytest
 
 from up42.asset import Asset
-from up42.order import Order
+from up42.order import Order, _translate_construct_parameters
```
Comment (javidq) [inline tests/test_order.py:6]: We don't test the hidden functions. Please parameterize the tests of order placement and estimation for the case of tasking

**Review by javidq — CHANGES_REQUESTED** [review]

File: tests/fixtures/fixtures_order.py [hunk 2]
```diff
@@ -93,3 +93,41 @@ def order_parameters():
...
+            "spectralProcessing": None,
+            "pixelCoding": None,
+            "radiometricProcessing": None,
+            "deliveredAs": None,
+        },
+    }
+
+
+@pytest.fixture(params=["catalog", "tasking"])
+def order_parameters_tasking_catalog(request, catalog_order_parameters, tasking_order_parameters):
```
Comment (javidq) [inline tests/fixtures/fixtures_order.py:131]: Just order_parameters, pytest will display params anyway

File: tests/test_catalog.py [hunk 3]
```diff
@@ -457,25 +457,32 @@ def test_estimate_order_from_catalog(order_parameters, requests_mock, auth_mock)
...
-    estimation = catalog_instance.estimate_order(order_parameters)
+    estimation = catalog_instance.estimate_order(catalog_order_parameters)
     assert isinstance(estimation, int)
     assert estimation == 100
 
 
-def test_order_from_catalog(order_parameters, order_mock, catalog_mock, requests_mock):
+def test_order_from_catalog(
+    catalog_order_parameters,
+    tasking_order_parameters,
```
Comment (javidq) [inline tests/test_catalog.py:467]: Why do we need these?

**Review by javidq — CHANGES_REQUESTED** [review]

File: tests/test_catalog.py (hunk 3 above)
Comment (andher1802) [reply tests/test_catalog.py]: You are right, I have added those to test separately and forgot to remove them at the end.

**Pull request merged**